Updated changelog (#617)
memgraph/docs: docs/changelog.md

### Major Features and Improvements

> - Added option to use `DISTINCT` with aggregation functions.

Reviewer: ```suggestion
- `DISTINCT` subclause can now be used with aggregation functions.
```

To do what? 
(we are trying to be a bit more user-oriented... so what does this enable me? )
Also: 

For example, query `COUNT(DISTINCT n.prop)` .... will do what?

> - `DISTINCT` subclause can now be used with aggregation functions.
>   Until now, if you wanted to use an aggregation function with distinct values, you had to write a query similar to this one `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)`. Now you can use the `DISTINCT` clause like in the following query,  `RETURN COUNT(DISTINCT n.prop)` . [#654](https://github.com/memgraph/memgraph/pull/665)

Reviewer: ```suggestion
- `DISTINCT` subclause can now be used with aggregation functions.
  Until now, if you wanted to use an aggregation function with distinct values, you had to write a query similar to this one `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)`. Now you can use the `DISTINCT` clause like in the following query, `RETURN COUNT(DISTINCT n.prop)` . [#654](https://github.com/memgraph/memgraph/pull/665)
```

>   Until now if you wanted to use aggregation function with distinct values you had to write a query for example `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)` while now with this new improvement query would look like this `RETURN COUNT(DISTINCT n.prop)` which is less complex and has better performance. [#654](https://github.com/memgraph/memgraph/pull/665)

Reviewer: ```suggestion
  Until now, if you wanted to use an aggregation function with distinct values, you had to write a query similar to this one `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)`. Now you can use the `DISTINCT` clause like in the following query,  `RETURN COUNT(DISTINCT n.prop)` . [#654](https://github.com/memgraph/memgraph/pull/665)
```
